content/en/platform/corda/5.2/deploying-operating/keys/_index.md

```diff
@@ -0,0 +1,196 @@
+title: "Managing Wrapping Keys"
+date: '2024-02-12'
+menu:
+  corda52:
+    identifier: corda52-cluster-wrapping-keys
+    parent: corda52-cluster
+    weight: 3025
+---
+
+# Managing Wrapping Keys
+
+As described in the [Architecture for Cluster Administrators]({{< relref "../../key-concepts/cluster-admin/_index.md#key-management" >}}) section, Corda keys are stored in the cluster and virtual node `Crypto` databases and encrypted at rest with wrapping keys.
+These managed wrapping keys are in turn wrapped by a master wrapping key.
+This section describes how Corda stores the master wrapping key and how to rotate master and managed wrapping keys. It contains the following:
+
+{{< toc >}}
+
+## Configuring the Master Wrapping Key
+
+The master wrapping key, or the information required to generate this key, must be stored and managed outside Corda.
+You can achieve this in one of the following ways:
+
+* Pass a passphrase and salt, to generate the master key, into the crypto worker processes. For more information, see [Default Secrets Service]({{< relref "../../deploying-operating/deployment/deploying/_index.md#default-secrets-service" >}}).
+* {{< enterprise-icon noMargin="true" >}} Store and manage the master in an external key management system for Corda to retrieve when required. For more information, see [External Secrets Service]({{< relref "../../deploying-operating/deployment/deploying/_index.md#external-secrets-service" >}}).
+
+Corda stores the master wrapping key settings in the `corda.crypto` configuration section.
+For information about the configuration fields in this section, see [corda.crypto]({{< relref "../config/fields/crypto.md" >}}).
+For information about how to retrieve the current values of a configuration section, see [Retrieving Current Configuration Values]({{< relref "../config/dynamic.md#retrieving-current-configuration-values" >}}).
+
+## Rotating Wrapping Keys
+
+R3 recommend changing the wrapping keys frequently to limit the probability of a compromised key being in use.
+Corda supports the following key rotation:
+
+* [Master Wrapping Key](#rotating-master-wrapping-key)
+* [Managed Wrapping Keys](#rotating-managed-wrapping-keys)
+
+### Rotating Master Wrapping Key
+
+Because the master key is not managed by Corda, you must first change the default key in the source system.
+Corda must then re-encrypt all of the keys that were wrapped with the old master key.
+To rotate the master wrapping key, do the following:
+
+1. Add the new key to the source environment. For more information, see [Configuring the Master Wrapping Key](#configuring-the-master-wrapping-key).
+2. Add the new key to the `corda.crypto` configuration section, setting it as the default key. The [Setting Configuration Fields Dynamically]({{< relref "../config/dynamic.md#setting-configuration-fields-dynamically" >}}) section describes how to update configuration values. For example, to add a new master key with the alias `master2`:
```

Comment:
Perhaps add "...  from now on all new managed wrapping keys will be encrypted with this new master key"